docs/components/input-controller/gamepad.md

```diff
@@ -97,7 +98,7 @@ If you have another controller that you want to use to control your robot, feel
 ## Troubleshooting
 
 - If you are not able to see a drop-down menu with the name of your controller appear in the **Control** tab, try specifying the `dev_file` attribute to match the exact path to your device.
-You can also try setting `auto_reconnect` to `True`.
+  You can also try setting `auto_reconnect` to `True`.
```

Comment:
Is this right? single item UL element is to be ... indented two whitespace characters? Would have expected `-` per other instances.

Reply:
This is a continuation of the previous item not a new one. So the indentation is to match that item